.craft.yml

```diff
@@ -24,6 +24,10 @@ targets:
   - name: npm
     id: '@sentry/replay'
     includeNames: /^sentry-replay-\d.*\.tgz$/
+  ## 1.6. OpenTelemetry package
+  - name: npm
+    id: '@sentry/opentelemetry'
```

Comment:
We prob also need to add this to the verdaccio config for the e2e tests.

Reply:
should this not automatically be picked up there? 🤔 based on the namespace...?

Reply:
No unfortunately not: 

https://github.com/getsentry/sentry-javascript/blob/cadeefe5c3a99b835a6c23f2c2f25b9b94a3640a/packages/e2e-tests/verdaccio-config/config.yaml#L29-L39

Reply:
damn, good catch! I'll actually add this here as well, is missing there so far: https://github.com/getsentry/sentry-javascript/blob/develop/docs/new-sdk-release-checklist.md